docs/guides/integration-javascript-client-side.md

```diff
@@ -326,18 +338,20 @@ window.__uid2.callbacks.push(async (eventType, payload) => {
         // }
         var advertising_token_to_use = payload.identity.advertising_token;
       } else {
-          if (__uid2.isLoginRequired()) {
+          if (__uid2.hasOptedOut()) {
+            // DII has opted out of UID2, no UID2 token will be generated
+          }
+          else if (__uid2.isLoginRequired()) {
             // Call one of the setIdentityFrom functions to generate a new UID2 token.
             // Add any retry logic around this call as required.
             await __uid2.setIdentityFromEmailHash(
                 emailHash,
-                clientSideConfig
-          );
```

Comment:
the indentations was wrong so actually there is syntax error. trying to fix it as well